modules/govuk/manifests/apps/ckan/cronjobs.pp

```diff
@@ -14,4 +21,13 @@
     hour           => '2',
   }
 
+  if $enable_solr_reindex {
+    govuk::apps::ckan::paster_cronjob { 'harvester reindex':
```

Comment:
Is there a situation where we'd want to turn this off quickly by flipping this flag?  If so should the `else` of this conditional be to remove the cron job rather than decline to add it?  If there's a distinction here.

Reply:
Good idea, I've pushed up a commit that allows `govuk::apps::ckan::paster_cronjob` to be `ensure => absent`.